playbook/app/pb_kits/playbook/pb_multiple_users/multiple_users.html.erb

```diff
@@ -4,11 +4,16 @@
     data: object.data,
     id: object.id) do %>
     <% object.users.take(object.display_count).each do |user| %>
-      <%= pb_rails("avatar", props: user.merge({size: "xs", classname: "pb_multiple_users_item", dark: object.dark}) ) %>
+      <%= pb_rails("avatar", props: user.merge({
+        classname: "pb_multiple_users_item",
+        dark: object.dark,
+        size: object.avatar_size(user[:size].present?),
+        })
+      )%>
     <% end %>
 
     <% if object.more_than_four %>
-      <div class="pb_multiple_users_item multiple_users_badge">
+      <div class="pb_multiple_users_item multiple_users_badge<%= (object.data[:size] == 'xxs') ? '_xxs' : '' %>">
```

Comment:
@thestephenmarshall @jasperfurniss This caught my eye. Is there a better approach?

Reply:
Sure. Why not make a whitelist and leverage `object.data[:size]`?